### PR TITLE
Actioncable and Actionpack documentation typos [ci skip]

### PR DESCRIPTION
**Pull request by tomkadwill**
### Summary

Improved documentation for Actionable and Acionpack

### CODE REVIEW

File: actionpack/lib/abstract_controller/helpers.rb [hunk 1]
```diff
@@ -157,7 +157,7 @@ def modules_for_helpers(args)
               mod_name.constantize
             rescue LoadError
               # dependencies.rb gives a similar error message but its wording is
-              # not as clear because it mentions autoloading. To the user all it
+              # not as clear because it mentions autoloading. To the user all that
```
Comment (vipulnsward) [inline actionpack/lib/abstract_controller/helpers.rb]: Lets keep this

File: actionpack/lib/action_controller/metal/conditional_get.rb [hunk 2]
```diff
@@ -13,9 +13,8 @@ module ConditionalGet
 
     module ClassMethods
       # Allows you to consider additional controller-wide information when generating an ETag.
-      # For example, if you serve pages tailored depending on who's logged in at the moment, you
-      # may want to add the current user id to be part of the ETag to prevent unauthorized displaying
-      # of cached pages.
+      # For example, if you serve pages tailored to the current logged in user, you may want
+      # the current user id to be part of the ETag to prevent unauthorized displaying of cached pages.
```
Comment (vipulnsward) [inline actionpack/lib/action_controller/metal/conditional_get.rb]: And this as well. Previous version one is simpler

File: actionpack/lib/action_controller/metal/data_streaming.rb [hunk 3]
```diff
@@ -25,14 +25,14 @@ module DataStreaming
       # * <tt>:filename</tt> - suggests a filename for the browser to use.
       #   Defaults to <tt>File.basename(path)</tt>.
       # * <tt>:type</tt> - specifies an HTTP content type.
-      #   You can specify either a string or a symbol for a registered type register with
-      #   <tt>Mime::Type.register</tt>, for example :json
-      #   If omitted, type will be guessed from the file extension specified in <tt>:filename</tt>.
-      #   If no content type is registered for the extension, default type 'application/octet-stream' will be used.
+      #   You can specify either a string or a symbol for a registered type.
+      #   register a type, for example :json, with <tt>Mime::Type.register</tt>.
```
Comment (vipulnsward) [inline actionpack/lib/action_controller/metal/data_streaming.rb]: `Register`

File: actionpack/lib/action_controller/metal/data_streaming.rb [hunk 4]
```diff
@@ -79,14 +79,15 @@ def send_file(path, options = {}) #:doc:
...
+      # the file name, and other things.
       #
       # Options:
       # * <tt>:filename</tt> - suggests a filename for the browser to use.
       # * <tt>:type</tt> - specifies an HTTP content type. Defaults to 'application/octet-stream'. You can specify
-      #   either a string or a symbol for a registered type register with <tt>Mime::Type.register</tt>, for example :json.
-      #   If omitted, type will be guessed from the file extension specified in <tt>:filename</tt>.
-      #   If no content type is registered for the extension, default type 'application/octet-stream' will be used.
+      #   either a string or a symbol for a registered type.
+      #   register a type, for example :json, with <tt>Mime::Type.register</tt>.
```
Comment (vipulnsward) [inline actionpack/lib/action_controller/metal/data_streaming.rb]: `Register`

Comment (vipulnsward) [inline actionpack/lib/action_controller/metal/data_streaming.rb]: Actually just drop `register`. The sentence goes like this-

`You can specify either a string or a symbol for a registered type with <tt>Mime::Type.register</tt>, for example :json.`

File: actionpack/lib/action_controller/metal/data_streaming.rb (hunk 3 above)
Comment (vipulnsward) [inline actionpack/lib/action_controller/metal/data_streaming.rb]: See below, just remove the `register` word from here.

**Comment by vipulnsward** [comment]
Thanks @tomkadwill , some changes and ready to merge. Please note GitHub doesn't notify when new changes are pushed, so please ping me after pushing new changes.

**Comment by tomkadwill** [comment]
Thanks @vipulnsward, I've updated the PR to include your suggestions.

**Comment by vipulnsward** [comment]
Thanks @tomkadwill !

**Pull request merged**